packages/byods/src/data-source-client/index.ts

```diff
@@ -7,6 +16,7 @@ import {HttpClient, ApiResponse} from '../http-client/types';
  */
 export default class DataSourceClient {
   private httpClient: HttpClient;
+  private timer: NodeJS.Timeout | null;
```

Comment:
does this support multiple data sources? if we call scheduleAutoRefresh will this gets overwritten?

Reply:
**_Need to discuss this with team_**